homeassistant/components/axis/manifest.json

```diff
@@ -3,11 +3,11 @@
   "name": "Axis",
   "config_flow": true,
   "documentation": "https://www.home-assistant.io/integrations/axis",
-  "requirements": ["axis==35"],
+  "requirements": ["axis==37"],
   "zeroconf": [
-    {"type":"_axis-video._tcp.local.","macaddress":"00408C*"},
-    {"type":"_axis-video._tcp.local.","macaddress":"ACCC8E*"},
-    {"type":"_axis-video._tcp.local.","macaddress":"B8A44F*"}
+    { "type": "_axis-video._tcp.local.", "macaddress": "00408C*" },
```

Comment:
It's not standard to have leading and trailing spaces in the beginning and end of the json object. Just put a space after the commas.

Reply:
It was black's doing, nothing I wanted changed personally

Reply:
Black formats this file with four space indent and without leading and trailing spaces in the objects when I run it specifically on this file. Prettier does add the leading and trailing spaces, though.

Reply:
Yeah, so let's not worry about this. It just looked weird to me.